blog/_posts/2016-03-28-4-ways-to-find-a-team.md

```diff
@@ -0,0 +1,62 @@
+
+If you don’t know someone in your community, you have four ways to find a teammate or a coach: Getting in touch with local Rails Girls Chapters or other similar initiatives, sending an email to our Rails Girls Summer of Code Google Group, tweeting or going to out Teams App.
+
+
+### Getting in Touch with Local Programming Initiatives
+
+![Getting in Touch with Local Programming Initiatives](/img/blog/2016/RGSoC-Initiatives-Find-a-Team.gif)
+
+One of the most effective ways to find a teammate or a coach in your city, is to getting in touch with a Rails Girls chapter, a study group or other initiatives like Black Girls Code or PHP Women.  
+
+Before reaching out to them, please take a look at the projects you want to apply and the programming languages of the project. This way, you will narrow the initiatives by programing language and you can focus on finding someone who has some knowledge on that language.
+
+
+### Rails Girls Summer of Code Google Group
+
+![Rails Girls Summer of Code Google Group](/img/blog/2016/RGSoC-Google-Group-Find-a-Team.gif)
+
+At the moment there are almost 400 people in this Google Group and we never know if someone there knows a person that could be interested to pair with you or be a coach of your team. 
+When you submit a new topic, please say what you are looking for and don’t forget to mention the city!
+
+
+### Twitter
+
+![Twitter and RGSoC](/img/blog/2016/RGSoC-Twitter-Search-Team.gif)
+
+A good way to reach to the community is through twitter. You have 140 characters so use them carefully: always mention what you are looking for and identify the city where you are going to be working on the Summer.  
+
+One of the most common mistakes we see is when people tweet asking for a teammate or a coach but they don’t mention the city. This makes it difficult for someone to find out about the place and because of that, they can’t help you by retweeting or getting in touch with you.  
+
+Another important thing is to mention us [@RailsGirlsSoC](https://twitter.com/RailsGirlsSoC
+) on the tweet, so that we can get a notification and retweet you. In case you have some space, use the hashtags #rgsoc or #rgsoc2016 to make it easier for people to find your tweets.
+
+
+### Rails Girls Summer of Code Teams App
+
+![Rails Girls Summer of Code Teams App](/img/blog/2016/RGSoC-Teams-App-Find-a-Team.gif)
+
+When you sign up to [RGSoC Teams App](teams.railsgirlssummerofcode.org), on the top right corner you will find our a link to our [Community](thttps://teams.railsgirlssummerofcode.org/users). There are over 800 people there and even though it is not a seamless process (please bare with us, we are working on it!), here you can filter people by their roles or interests. This means you can look for students that want to find a team or a developer that wants to help out as a coach. Then, you can sort the info by country or city.
```

Comment:
this link needs to be changed to `http://teams.railsgirlssummerofcode.org` for it to work properly :)